### PR TITLE
Fix ClassCastException in Eclipse WTP

### PR DESCRIPTION
**Pull request by reinsch82**
- #24967

### CODE REVIEW

File: subprojects/ide/src/integTest/groovy/org/gradle/plugins/ide/eclipse/EclipseWtpIntegrationTest.groovy [hunk 1]
```diff
@@ -37,10 +37,10 @@ dependencies {
         libsDir.mkdir()
         libsDir.createFile("foo.jar")
 
-        // when
+        when:
```
Comment (donat) [inline subprojects/ide/src/integTest/groovy/org/gradle/plugins/ide/eclipse/EclipseWtpIntegrationTest.groovy:40]: ❌ Even though it's written in Groovy, this still seems like a JUnit test that does not have the `when:` and `then:` constructs.

**Review by donat — CHANGES_REQUESTED** [review]
LGTM. I have only one concern regarding test coverage.

**Comment by reinsch82** [comment]
@bot-gradle test this

**Review by donat — APPROVED** [review]

**Comment by reinsch82** [comment]
@bot-gradle test and merge

**Pull request merged**